src/components/EmailForm.tsx

```diff
@@ -30,19 +31,21 @@ export default function ({
           event.code === 'Enter' && emailIsValid ? onSubmit(email) : undefined
         }
       />
-      <Button
-        gradientFont={submitType !== 'primary'}
-        loading={loading}
-        loadingOverflow
-        fullWidth
-        center
-        small={submitType !== 'primary'}
-        type={submitType}
-        disabled={!emailIsValid}
-        onClick={() => onSubmit(email)}
-      >
-        {submitText}
-      </Button>
+      <GradientButton>
```

Comment:
Maybe we should name it `GradientBorder` instead?

Reply:
I think you're right, it'll have more meaning. I'll rename it, ty